src/mjolnir/shortcutbuilder.cc

```diff
@@ -29,6 +29,9 @@ using namespace valhalla::mjolnir;
 
 namespace {
 
+// total_shortcut_count and total_superseded_edges
+using shortcut_stats = std::pair<uint32_t, uint32_t>;
```

Comment:
nit this typedef is only used for the return type. in the functions that return it and in the function that accumulates it its not used instead separate variables are used. we could just drop the typedef and return pair explicitly